docker-compose.yml

```diff
@@ -5,20 +5,19 @@ services:
   # officially supported).
   # To run in macOS M1, set FLEET_MYSQL_IMAGE=arm64v8/mysql:oracle FLEET_MYSQL_PLATFORM=linux/arm64/v8
   mysql:
-    image: ${FLEET_MYSQL_IMAGE:-mysql:5.7}
+    image: ${FLEET_MYSQL_IMAGE:-mysql:5.7.21}
```

Comment:
Changing this because this is what we currently use in CI:
https://github.com/fleetdm/fleet/blob/caa27322fecacac8d9d8d73591f560401f46f0c6/.github/workflows/test-go.yaml#L48

My test was working locally but failing in CI due to this difference.